### PR TITLE
Added Key Store

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
- Validated the addition of the new keystore "sankey_widget_sample_data" containing "Sample data" to the solution pack.
- Validated that the dashboard configuration is updated to display sample data instead of live data by default.

### CODE REVIEW

File: dashboards/SOC Alert Flow (Sankey).json [hunk 1]
```diff
@@ -81,7 +81,24 @@
...
+                                    "query": {
+                                        "sort": [],
+                                        "limit": 30,
+                                        "logic": "AND",
+                                        "filters": [
+                                            {
+                                                "field": "key",
+                                                "operator": "eq",
+                                                "_operator": "eq",
+                                                "value": "sankey_widget_sample_data",
```
Comment (cs-suyog-jinturkar) [inline dashboards/SOC Alert Flow (Sankey).json]: widget_sankey_sample_data

File: records/keys/keys0001.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
+[
+    {
+        "@type": "Key",
+        "key": "sankey_widget_sample_data",
```
Comment (cs-suyog-jinturkar) [inline records/keys/keys0001.json]: widget_sankey_sample_data

File: dashboards/SOC Alert Flow (Sankey).json (hunk 1 above)
Comment (cs-rohit-khune) [reply dashboards/SOC Alert Flow (Sankey).json]: Done changes. Please check

File: records/keys/keys0001.json (hunk 2 above)
Comment (cs-rohit-khune) [reply records/keys/keys0001.json]: Done

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**